app/models/lesson_study.rb

```diff
@@ -0,0 +1,19 @@
+class LessonStudy < ApplicationRecord
+  belongs_to :course
```

Comment:
All of this should go inside a migration file, which you can generate with:
`rails generate migration createLessonSchedule`